auth/src/main/AndroidManifest.xml

```diff
@@ -19,6 +19,10 @@
             android:name="com.facebook.sdk.ApplicationId"
             android:value="@string/facebook_application_id"/>
 
+        <meta-data
```

Comment:
@SUPERCILEX Could we be inadvertently disabling something in developer's apps?  What if they have a conflicting value in their `AndroidManifest`?